torch/utils/data/dataloader.py

```diff
@@ -44,6 +44,8 @@
 ]
 
 
+T_co = TypeVar('T_co', covariant=True)
+T = TypeVar('T')
```

Comment:
This is being fixed https://github.com/pytorch/data/pull/1282
After that PR landed, you can reinstall the latest torchdata nightly to avoid manually changing it here.